src/quotes.rs

```diff
@@ -397,106 +397,214 @@ impl YQuoteSummary {
-    #[serde(rename = "financialData")]
-    pub financial_data: FinancialData,
+    pub asset_profile: Option<AssetProfile>,
+    pub summary_detail: Option<SummaryDetail>,
+    pub default_key_statistics: Option<DefaultKeyStatistics>,
+    pub quote_type: Option<QuoteType>,
+    pub financial_data: Option<FinancialData>,
 }
 
 #[derive(Deserialize, Debug)]
+#[serde(rename_all = "camelCase")]
 pub struct AssetProfile {
-    pub address1: String,
-    pub city: String,
-    pub state: String,
-    pub zip: String,
-    pub country: String,
-    pub phone: String,
-    pub website: String,
-    pub industry: String,
-    pub sector: String,
-    #[serde(rename = "longBusinessSummary")]
-    pub long_business_summary: String,
-    #[serde(rename = "fullTimeEmployees")]
-    pub full_time_employees: u32,
-    #[serde(rename = "companyOfficers")]
+    pub address1: Option<String>,
+    pub city: Option<String>,
+    pub state: Option<String>,
+    pub zip: Option<String>,
+    pub country: Option<String>,
+    pub phone: Option<String>,
+    pub website: Option<String>,
+    pub industry: Option<String>,
+    pub sector: Option<String>,
+    pub long_business_summary: Option<String>,
+    pub full_time_employees: Option<u32>,
     pub company_officers: Vec<CompanyOfficer>,
+    pub audit_risk: Option<u16>,
+    pub board_risk: Option<u16>,
+    pub compensation_risk: Option<u16>,
+    pub share_holder_rights_risk: Option<u16>,
+    pub overall_risk: Option<u16>,
+    pub governance_epoch_date: Option<u32>,
+    pub compensation_as_of_epoch_date: Option<u32>,
+    pub ir_website: Option<String>,
+    pub max_age: Option<u32>,
 }
 
 #[derive(Deserialize, Debug)]
+#[serde(rename_all = "camelCase")]
 pub struct CompanyOfficer {
     pub name: String,
     pub age: Option<u32>,
     pub title: String,
-    #[serde(rename = "yearBorn")]
     pub year_born: Option<u32>,
-    #[serde(rename = "fiscalYear")]
     pub fiscal_year: u32,
-    #[serde(rename = "totalPay")]
     pub total_pay: Option<ValueWrapper>,
 }
 
 #[derive(Deserialize, Debug)]
+#[serde(rename_all = "camelCase")]
 pub struct ValueWrapper {
     pub raw: Option<u64>,
     pub fmt: Option<String>,
-    #[serde(rename = "longFmt")]
     pub long_fmt: Option<String>,
 }
 
 #[derive(Deserialize, Debug)]
+#[serde(rename_all = "camelCase")]
 pub struct SummaryDetail {
-    #[serde(rename = "previousClose")]
-    pub previous_close: f64,
-    pub open: f64,
-    #[serde(rename = "dayLow")]
-    pub day_low: f64,
-    #[serde(rename = "dayHigh")]
-    pub day_high: f64,
-    #[serde(rename = "regularMarketVolume")]
-    pub regular_market_volume: u64,
-    #[serde(rename = "marketCap")]
-    pub market_cap: u64,
-    pub currency: String,
+    pub max_age: Option<i64>,
+    pub price_hint: Option<i64>,
+    pub previous_close: Option<f64>,
+    pub open: Option<f64>,
+    pub day_low: Option<f64>,
+    pub day_high: Option<f64>,
+    pub regular_market_previous_close: f64,
```

Comment:
I am assuming this should also be optional?
```suggestion
    pub regular_market_previous_close: Option<f64>,
```

Reply:
You're Goddamn Right